client/src/partials/vouchers/toolkit/convention_payment.js

```diff
@@ -0,0 +1,176 @@
+angular.module('bhima.controllers')
+.controller('ConventionPaymentKitController', ConventionPaymentKitController);
+
+// DI definition
+ConventionPaymentKitController.$inject = [
+  '$uibModalInstance', 'DebtorGroupService', 'NotifyService',
+  'CashboxService', 'SessionService', 'data',
+  'AccountStoreService', 'DebtorService', 'PatientInvoiceService'
+];
+
+// Import transaction rows for a convention payment
+function ConventionPaymentKitController(Instance, DebtorGroup, Notify, Cashbox, Session, Data, AccountStore, Debtors, Invoices) {
+  var vm = this;
+
+  // global variables
+  vm.enterprise = Session.enterprise;
+  vm.gridOptions = {};
+  vm.tool = Data;
+
+  // expose to the view
+  vm.selectGroupInvoices = selectGroupInvoices;
+  vm.close = Instance.close;
+  vm.import = submit;
+
+  // accounts from store
+  AccountStore.accounts().then(function (data) {
+    vm.accounts = data;
+  });
+
+  // debtors from store
+  // FIXME: need auto refresh
+   Debtors.store.then(function (data) {
+     vm.debtors = data;
+   });
+
+  //  optimization with `Store` will be well
+   Invoices.read()
+   .then(function (data) {
+     vm.invoices = data;
+   });
```

Comment:
All of these should have `Notify.handleError` to make sure the errors in network connections are reported.